File: backend/src/main/java/com/campus/Campus_Connect/features/map/dto/response/PresenceResponse.java
package com.campus.Campus_Connect.features.map.dto.response;

import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PresenceResponse {

    private Double latitude;

    private Double longitude;

    private Boolean insideCampus;

    private String visibility;

    private LocalDateTime lastUpdated;
}

File: android-app/app/src/main/java/com/example/campusconnect/feature/map/data/remote/response/PresenceResponse.kt
package com.example.campusconnect.feature.map.data.remote.response


data class PresenceResponse(

    val latitude: Double,

    val longitude: Double,

    val insideCampus: Boolean,

    val visibility: String,

    val lastUpdated: String,

    val gender: String? = null
)